### ISSUE TITLE
TVSHows will not play, blank screen

### ISSUE THREAD

**Issue by thetravellor** [CastagnaIT/plugin.video.netflix#837]
#### Your Environment
- Netflix add-on version: libdev+jojo+asciidisco+caphm+castagnait - 1.80
- Operating system version/name: PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs

Kodi:
ii  kodi                                  2:18.7-1~buster                        all          Kodi Media Center (arch-independent data package)
ii  kodi-bin                              2:18.7-1~buster                        armhf        Kodi Media Center (binary data package)
ii  kodi-inputstream-adaptive             2.4.4-1~buster                         armhf        adaptive inputstream addon for Kodi
ii  kodi-inputstream-rtmp                 2.0.6-1~buster                         armhf        Rtmp inputstream addon for Kodi
ii  kodi-peripheral-joystick              1.4.8-1~buster                         armhf        Joystick peripheral addon for Kodi
ii  kodi-pvr-iptvsimple                   3.8.8-1~buster                         armhf        Simple IPTV PVR for Kodi

- Device model:  Raspberry Pi 4, 8GB model

Used Operating system:
* [ ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [x] Raspberry-Pi
* [ ] Windows

#### Expected behavior

I should be able to play a TV show in Kodi

the addon cannot play any TV shows at all, only movies
#### Actual behavior

After selecting a TV show episode in the Netflix addon, I instead get a blank blue screen

#### Steps to reproduce the behavior

1. Open Kodi
2. Launch addon
3. Wait for background services to start
4. Select profile
5. Browse to any TV show 
6. Select it, and then nothing.

### Debug log

The debug log can be found from this link:

[https://github.com/thetravellor/My-Logs/blob/master/kodi.log](url)

### Additional context or screenshots (if appropriate)

Pictures
[https://imgur.com/a/g1gZiZE](url)

#### Installation

I am not sure how to respond: 

* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source 

I followed the instructions form this blog.
[https://pimylifeup.com/raspberry-pi-netflix/](url)
And got the addon from here: wget https://github.com/castagnait/repository.castagnait/raw/master/repository.castagnait-1.0.1.zip

#### Other information

I can play movies no problem, only TV shows will not work

**Comment by CastagnaIT** [comment]
i do not see playback info in the log, give a log after try to play a video with black screen

**Comment by thetravellor** [comment]
Hello Stefano,

 

This issue is that the addon is not returning the list of TV shows in the season (or if it is, Kodi is not displaying them)

 

Essentially I started up Kodi, immediately went to where the defect is, reproduced it, then pressed ESC until I got to the home screen and shutdown Kodi. I had thought the logfile 

I sent contained the right data, I think this represents the situation:

 

2020-09-02 22:12:34.211 T:2973080672   DEBUG: ProcessMouse: trying mouse action leftclick

2020-09-02 22:12:34.212 T:2973080672   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.netflix/directory/show/81002370/)

2020-09-02 22:12:34.212 T:2973080672   DEBUG:   ParentPath = [plugin://plugin.video.netflix/directory/video_list/mostWatched/mostWatched/]

2020-09-02 22:12:34.213 T:2413936848   DEBUG: Thread waiting start, auto delete: false

2020-09-02 22:12:34.222 T:2963271888   DEBUG: libinput: event17 - debounce state: DEBOUNCE_STATE_RELEASED → DEBOUNCE_EVENT_TIMEOUT → DEBOUNCE_STATE_IS_UP

2020-09-02 22:12:34.232 T:2345337040   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.netflix] took 20 ms

2020-09-02 22:12:34.233 T:2413936848   DEBUG: StartScript - calling plugin Netflix('plugin://plugin.video.netflix/directory/show/81002370/','1','','resume:false')

2020-09-02 22:12:34.233 T:2413936848   DEBUG: GetLanguageInvoker - Reusing LanguageInvokerThread 2 for script /home/pi/.kodi/addons/plugin.video.netflix/addon.py

2020-09-02 22:12:34.233 T:2388623568    INFO: initializing python engine.

2020-09-02 22:12:34.233 T:2388623568   DEBUG: CPythonInvoker(2, /home/pi/.kodi/addons/plugin.video.netflix/addon.py): start processing

2020-09-02 22:12:34.234 T:2388623568   DEBUG: CPythonInvoker(2, /home/pi/.kodi/addons/plugin.video.netflix/addon.py): entering source directory /home/pi/.kodi/addons/plugin.video.netflix

2020-09-02 22:12:34.234 T:2388623568   DEBUG: CPythonInvoker(2, /home/pi/.kodi/addons/plugin.video.netflix/addon.py): instantiating addon using automatically obtained id of "plugin.video.netflix" dependent on version 2.26.0 of the xbmc.python api

2020-09-02 22:12:34.239 T:2388623568   DEBUG: CAddonSettings[plugin.video.netflix]: loading setting definitions

2020-09-02 22:12:34.239 T:2388623568   DEBUG: CAddonSettings[plugin.video.netflix]: trying to load setting definitions from old format...

2020-09-02 22:12:34.246 T:2388623568   DEBUG: CAddonSettings[plugin.video.netflix]: loading setting values

2020-09-02 22:12:34.251 T:2388623568    INFO: [plugin.video.netflix (1)] Started (Version 1.8.0)

2020-09-02 22:12:34.251 T:2388623568    INFO: [plugin.video.netflix (1)] URL is ParseResult(scheme='plugin', netloc='plugin.video.netflix', path='/directory/show/81002370/', params='', query='', fragment='')

2020-09-02 22:12:34.313 T:2973080672   DEBUG: ------ Window Init (DialogBusy.xml) ------

2020-09-02 22:12:34.322 T:2388623568   DEBUG: [plugin.video.netflix (1)] Routing navigation request

2020-09-02 22:12:34.322 T:2388623568   DEBUG: [plugin.video.netflix (1)] Initializing "Directory" with params: {}

2020-09-02 22:12:34.322 T:2388623568   DEBUG: [plugin.video.netflix (1)] Invoking action: show

2020-09-02 22:12:34.323 T:2388623568   DEBUG: [plugin.video.netflix (1)] Handling AddonSignals IPC call to get_seasons

2020-09-02 22:12:34.399 T:2405544144   DEBUG: [plugin.video.netflix (0)] Requesting the seasons list for show show_81002370

2020-09-02 22:12:34.400 T:2405544144   DEBUG: [plugin.video.netflix (0)] Executing path request: [["videos", "81002370", "seasonList", {"to": 44, "from": 0}, "summary"], ["videos", "81002370", "title"], ["videos", "81002370", "boxarts", ["_665x375", "_1920x1080", "_342x684"], "jpg"], ["videos", "81002370", "interestingMoment", ["_665x375", "_1920x1080"], "jpg"], ["videos", "81002370", "artWorkByType", "LOGO_BRANDED_HORIZONTAL", "_550x124", "png"], ["videos", "81002370", "storyArt", "_665x375", "jpg"]]

2020-09-02 22:12:34.417 T:2405544144   DEBUG: [plugin.video.netflix (0)] Executing POST request to https://www.netflix.com/api/shakti/vc5101a81/pathEvaluator

2020-09-02 22:12:34.686 T:2405544144   DEBUG: [plugin.video.netflix (0)] Request took 0.080145s

2020-09-02 22:12:34.686 T:2405544144   DEBUG: [plugin.video.netflix (0)] Request returned status code 200

2020-09-02 22:12:34.726 T:2413936848   DEBUG: Thread waiting 2413936848 terminating

2020-09-02 22:12:34.726 T:2388623568    INFO: CPythonInvoker(2, /home/pi/.kodi/addons/plugin.video.netflix/addon.py): script successfully run

2020-09-02 22:12:34.744 T:2973080672   DEBUG: ------ Window Deinit (DialogBusy.xml) ------

 

Does that help, if not, I can try again. I didn’t have component logging on, should I?

 

Stephen
